pkg/tests/testinfra/testinfra.go

```diff
@@ -495,9 +495,9 @@ func CreateGrafDir(t *testing.T, opts GrafanaOpts) (string, string) {
 	require.NoError(t, err)
 	_, err = dbSection.NewKey("query_retries", fmt.Sprintf("%d", queryRetries))
 	require.NoError(t, err)
-	_, err = dbSection.NewKey("max_open_conn", "2")
+	_, err = dbSection.NewKey("max_open_conn", "20")
```

Comment:
?? Why this change ?

Reply:
There are some tests spawning entire Grafana, with various features enabled. After some digging I’ve found that some of such tests get stuck when creating users or datasources, because they didn’t have enough DB connections. I suspect it’s related to transaction was aborted due to a concurrent modification errors I see in the logs. I’ve bumped number of connections to unstuck the tests (it helped), but they still don’t pass and need more work. Example of such tests are `TestIntegrationCreateCorrelation` (and other `TestIntegration*Correlation` tests).

Also using max 2 db connections seems very low in general.

Reply:
As discussed, increasing just for spanner 👍